cmd/containerd-stress/size.go

```diff
@@ -18,27 +18,34 @@ package main
 
 import (
 	"os"
-	"path/filepath"
 
 	"github.com/sirupsen/logrus"
 )
 
+const defaultPath = "/usr/local/bin"
```

Comment:
/usr/local/bin/ ?

Reply:
Sorry, I was reading the code and forgot to give a wip.